### PR TITLE
[FR] Cleanup `Gameplay\Grade`, `History_of_osu!\April_Fools`, `Online_rankings`, `Performance_troubleshooting`, `Reporting_bad_behaviour`, `Abuse`, `Handling_foul_play`

### PR DESCRIPTION
**Pull request by DoryanR**
Cleanup of the following wiki pages:

- `Gameplay\Grade`
- `History_of_osu!\April_Fools`
- `History_of_osu!\Online_rankings`
- `Performance_troubleshooting`
- `Reporting_bad_behaviour\Abuse`
- `Reporting_bad_behaviour`
- `Reporting_bad_behaviour\Handling_foul_play`
- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Reporting_bad_behaviour/Handling_foul_play/fr.md [hunk 1]
```diff
@@ -1,40 +1,36 @@
...
 
-Il s'agit d'une règle à un coup. Dès leur première infraction, les tricheurs seront soumis à des restrictions pour une durée indéterminée. L'[Équipe de support](/wiki/People/The_Team/Account_support_team) peut décider de lever la restriction si l'appel est accepté.
+Il s'agit d'une règle "à un coup". Dès leur première infraction, les tricheurs seront soumis à des restrictions pour une durée indéterminée. L'[Équipe de support](/wiki/People/The_Team/Account_support_team) peut décider de lever la restriction si l'appel est accepté.
 
 ## J'ai été restreint pour avoir triché, mais je n'ai pas triché !
 
 osu! utilise des mesures automatisées combinées à une confirmation manuelle. Si vous ne savez pas pourquoi vous êtes restreint, contactez [accounts@ppy.sh](mailto:accounts@ppy.sh).
 
-## Prouvez que je/ils ont triché ! Dis-nous comment vous le savez !
+## Prouvez que je/ils ont triché ! Dites-nous comment vous le savez !
```
Comment (Biokax) [inline wiki/Reporting_bad_behaviour/Handling_foul_play/fr.md:27]:
```suggestion
## Prouvez que j'ai/qu'ils ont triché(s) ! Dites-nous comment vous le savez !
```

File: wiki/Reporting_bad_behaviour/fr.md [hunk 2]
```diff
@@ -1,9 +1,3 @@
----
-outdated: true
-outdated_since: 2bad0ad3bfcdc272e5832ecf6d29bd61fcc6c23a
-no_native_review: true
----
-
 # Signaler un mauvais comportement
 
 osu! a [une équipe de modérateurs](/wiki/People/The_Team/Global_Moderation_Team) qui s'efforce de garder le jeu et son écosystème propres et sûrs. Si vous constatez un comportement contraire aux règles ou un contenu inapproprié, signalez-le en utilisant l'une des méthodes décrites ci-dessous. Vous pouvez également contacter directement l'un des modérateurs en ligne si votre report n'a pas été traité correctement dans un délai raisonnable, mais uniquement en dernier recours.
```
Comment (Biokax) [inline wiki/Reporting_bad_behaviour/fr.md:3]:
```suggestion
osu! a [une équipe de modérateurs](/wiki/People/The_Team/Global_Moderation_Team) qui s'efforcent de garder le jeu et son écosystème propres et sûrs. Si vous constatez un comportement contraire aux règles ou un contenu inapproprié, signalez-le en utilisant l'une des méthodes décrites ci-dessous. Vous pouvez également contacter directement l'un des modérateurs en ligne si votre report n'a pas été traité correctement dans un délai raisonnable, mais uniquement en dernier recours.
```

**Review by Biokax — CHANGES_REQUESTED** [review]

File: wiki/Performance_troubleshooting/fr.md [hunk 3]
```diff
@@ -12,24 +8,24 @@ Malheureusement, en raison de l'étendue du matériel sur lequel fonctionne osu!
...
+La piste audio des beatmaps revient en arrière ou s'interrompt. Lorsque le jeu lag, vous pouvez l'entendre dans l'audio. Lors du diagnostic ou du signalement, il peut être utile de :
 
-- Vérifiez si cela se produit sur tous les périphériques audio disponibles. Si vous utilisez un casque USB, essayez plutôt de vous connecter via la sortie analogique de votre PC.
-- Vérifiez si l'activation du "Mode de compatibilité audio" dans les paramètres est utile.
+- Vérifier si cela se produit sur tous les périphériques audio disponibles. Si vous utilisez un casque USB, essayez plutôt de vous connecter via la sortie analogique de votre PC.
+- Vérifier si l'activation du paramètre "Mode de compatibilité audio" dans les options vous aide.
 
 ### Spike frame
 
 Pendant le jeu, tout ce qui s'affiche à l'écran s'arrête pendant une durée suffisamment longue pour nuire à l'expérience de jeu. Cela peut se produire plus d'une fois, de façon aléatoire ou à intervalles réguliers. Lors du diagnostic ou du signalement, il peut être utile de :
```
Comment (Ganondorf-1) [inline wiki/Performance_troubleshooting/fr.md:20]:
```suggestion
Lorsque vous êtes en train de jouer, tout ce qui s'affiche à l'écran s'arrête pendant une durée suffisamment longue pour nuire à l'expérience de jeu. Cela peut se produire plus d'une fois, de façon aléatoire ou à intervalles réguliers. Lors du diagnostic ou du signalement, il peut être utile de :
```

File: wiki/Performance_troubleshooting/fr.md [hunk 4]
```diff
@@ -12,24 +8,24 @@ Malheureusement, en raison de l'étendue du matériel sur lequel fonctionne osu!
...
 
 - Comptez combien de fois cela se produit dans une session de jeu moyenne.
```
Comment (Ganondorf-1) [inline wiki/Performance_troubleshooting/fr.md:22]:
```suggestion
- Compter combien de fois cela se produit dans une session de jeu moyenne.
```

File: wiki/Performance_troubleshooting/fr.md [hunk 5]
```diff
@@ -12,24 +8,24 @@ Malheureusement, en raison de l'étendue du matériel sur lequel fonctionne osu!
...
-- Vérifiez si cela se produit toujours après un redémarrage du PC.
-- Essayez de fermer toute application qui affiche des superpositions sur le jeu (Discord, Steam, NVIDIA Shadowplay, etc.).
-- Assurez-vous que le [mode de jeu](https://www.windowscentral.com/how-enable-disable-game-mode-windows-10) est activé. Cela permet à Windows de fournir des cœurs et des ressources dédiés à osu!
-- Ouvrez le gestionnaire des tâches et classez la liste (sous Détails) par CPU. Recherchez toute utilisation élevée du CPU lorsque des pics se produisent, par exemple par un scanner de virus ou de logiciels malveillants en arrière-plan. Vous pouvez exécuter osu! en mode fenêtré ou laisser le gestionnaire de tâches sur un deuxième écran pour diagnostiquer cela pendant que vous jouez ou que vous exécutez une beatmap en auto.
+- Vérifier si cela se produit toujours.
+- Vérifier la fréquence à laquelle cela se produit, et si c'est à un intervalle régulier.
+- Vérifier si cela se produit toujours après un redémarrage du PC.
+- Essayer de fermer toute application qui affiche des superpositions en jeu (Discord, Steam, NVIDIA Shadowplay, etc.).
```
Comment (Ganondorf-1) [inline wiki/Performance_troubleshooting/fr.md:26]:
```suggestion
- Essayer de fermer toute application qui affiche des overlays en jeu (Discord, Steam, NVIDIA Shadowplay, etc.).
```

File: wiki/Performance_troubleshooting/fr.md [hunk 6]
```diff
@@ -12,24 +8,24 @@ Malheureusement, en raison de l'étendue du matériel sur lequel fonctionne osu!
...
+- Vous assurer que le [mode jeu](https://www.windowscentral.com/how-enable-disable-game-mode-windows-10) est activé. Cela permet à Windows de fournir des cœurs et des ressources dédiés à osu!
```
Comment (Ganondorf-1) [inline wiki/Performance_troubleshooting/fr.md:27]:
```suggestion
- Vous assurer que le [mode jeu](https://www.windowscentral.com/how-enable-disable-game-mode-windows-10) est activé. Cela permet à Windows de fournir des cœurs et des ressources dédiés à osu!.
```

File: wiki/Performance_troubleshooting/fr.md [hunk 7]
```diff
@@ -47,35 +43,35 @@ Pendant le jeu, un ou plusieurs périphériques d'entrée cessent de répondre.
...
-- Essayez de changer le limiteur d'images et voyez si cela améliore la situation.
-- Assurez-vous que vous exécutez osu! en mode plein écran exclusif (et non sans bordure). Cela améliorera toujours les performances.
-- Assurez-vous que le [mode de jeu](https://www.windowscentral.com/how-enable-disable-game-mode-windows-10) est activé. Il permet à Windows de fournir des cœurs et des ressources dédiés à osu!
-- Ouvrez le gestionnaire des tâches et classez la liste (sous Détails) par CPU. Recherchez toute utilisation élevée du processeur pendant les périodes de faible fréquence d'images, par exemple par un scanner de virus ou de logiciels malveillants en arrière-plan. Vous pouvez exécuter osu! en mode fenêtré ou laisser le gestionnaire de tâches sur un deuxième écran pour diagnostiquer cela pendant que vous jouez ou que vous exécutez une beatmap en autoplay.
+- Vérifier si cela se produit toujours après un redémarrage du PC.
+- Essayer de fermer toute application qui affiche des superpositions sur le jeu (Discord, Steam, NVIDIA Shadowplay, etc.).
```
Comment (Ganondorf-1) [inline wiki/Performance_troubleshooting/fr.md:49]:
```suggestion
- Essayer de fermer toute application qui affiche des overlays sur le jeu (Discord, Steam, NVIDIA Shadowplay, etc.).
```

File: wiki/Performance_troubleshooting/fr.md [hunk 8]
```diff
@@ -47,35 +43,35 @@ Pendant le jeu, un ou plusieurs périphériques d'entrée cessent de répondre.
...
+- Vous assurer que le [mode de jeu](https://www.windowscentral.com/how-enable-disable-game-mode-windows-10) est activé. Il permet à Windows de fournir des cœurs et des ressources dédiés à osu!
+- Ouvrir le gestionnaire des tâches et classer la liste (sous Détails) par CPU. Recherchez toute utilisation élevée du CPU lorsque des pics se produisent, par exemple par un scan de virus ou de logiciels malveillants en arrière-plan. Vous pouvez exécuter osu! en mode fenêtré ou laisser le gestionnaire de tâches sur un deuxième écran pour diagnostiquer cela pendant que vous jouez ou que vous exécutez une beatmap avec le mod Auto.
 
 ## Limiteurs d'images
 
-osu! offre plusieurs choix de limiteurs de frame. Chacun d'entre eux présente des avantages et des inconvénients. Notez que quel que soit le réglage, osu! limitera la fréquence d'images dans les menus pour économiser l'utilisation du CPU.
+osu! offre plusieurs choix de limiteurs de frame. Chacun d'entre eux présente des avantages et des inconvénients. Notez que quel que soit le réglage, osu! limitera la fréquence d'images dans les menus pour économiser l'utilisation CPU.
```
Comment (Ganondorf-1) [inline wiki/Performance_troubleshooting/fr.md:58]:
```suggestion
osu! offre plusieurs choix de limiteurs d'images. Chacun d'entre eux présente des avantages et des inconvénients. Notez que quel que soit le réglage, osu! limitera la fréquence d'images dans les menus pour économiser l'utilisation CPU.
```

File: wiki/Performance_troubleshooting/fr.md [hunk 9]
```diff
@@ -47,35 +43,35 @@ Pendant le jeu, un ou plusieurs périphériques d'entrée cessent de répondre.
...
 
-Généralement non recommandé pour les écrans 60Hz. Il peut devenir plus utile à des taux de rafraîchissement supérieurs à 120Hz, mais avec les moniteurs GSync et FreeSync, l'obligation d'utiliser VSync pour éviter le déchirement est levée.
+Généralement non recommandé pour les écrans 60Hz. Il peut devenir plus utile à des taux de rafraîchissement supérieurs à 120Hz, mais avec les moniteurs GSync (NVIDIA) et FreeSync (AMD), l'obligation d'utiliser VSync pour éviter le déchirement est levée.
 
 ### Économie d'énergie
 
-Ce limiteur d'images vise à économiser l'énergie tout en offrant une expérience de jeu cohérente. Il fonctionnera à une fréquence de rafraîchissement du moniteur de 2x et tentera de maintenir un rythme d'images constant.
+Ce limiteur d'images vise à économiser l'énergie tout en offrant une expérience de jeu cohérente. Il fonctionnera à la fréquence de rafraîchissement du moniteur 2 fois inférieure et tentera de maintenir un rythme d'images constant.
```
Comment (Ganondorf-1) [inline wiki/Performance_troubleshooting/fr.md:68]:
```suggestion
Ce limiteur d'images vise à économiser l'énergie tout en offrant une expérience de jeu constante. Il fonctionnera à la fréquence de rafraîchissement du moniteur 2 fois inférieure et tentera de maintenir un rythme d'images constant.
```

File: wiki/Performance_troubleshooting/fr.md [hunk 10]
```diff
@@ -47,35 +43,35 @@ Pendant le jeu, un ou plusieurs périphériques d'entrée cessent de répondre.
...
 
 Recommandé pour les PC plus anciens ou pour économiser la batterie d'un ordinateur portable.
 
 ### Optimal
 
-Ce limiteur de frame tente de maintenir un taux de frame 8x votre taux de rafraîchissement, avec un maximum de 960 FPS. C'est le limiteur d'images que nous recommandons aux utilisateurs qui ne veulent pas pousser leur CPU/GPU à l'extrême, tout en maintenant des niveaux élevés de performance résultant d'une faible latence d'entrée.
+Ce limiteur de frame tente de maintenir un taux de frame 8 fois supérieure à votre taux de rafraîchissement, avec un maximum de 960 FPS. C'est le limiteur d'images que nous recommandons aux utilisateurs qui ne veulent pas pousser leur CPU/GPU à l'extrême, tout en maintenant des niveaux élevés de performance résultant d'une faible latence d'entrée.
```
Comment (Ganondorf-1) [inline wiki/Performance_troubleshooting/fr.md:74]:
```suggestion
Ce limiteur d'images tente de maintenir un taux d'images 8 fois supérieure à votre taux de rafraîchissement, avec un maximum de 960 FPS. C'est le limiteur d'images que nous recommandons aux utilisateurs qui ne veulent pas pousser leur CPU/GPU à l'extrême, tout en maintenant des niveaux élevés de performance résultant d'une faible latence d'entrée.
```

File: wiki/Reporting_bad_behaviour/Abuse/fr.md [hunk 11]
```diff
@@ -61,7 +57,7 @@ Cliquez sur les trois points en haut à droite du tweet problématique et clique
 
 Ensuite, bloquez ou mettez en sourdine l'utilisateur en question en cliquant sur les trois points en haut à droite de son tweet (si vous utilisez le site web ou le client mobile). Consultez [le guide de Twitter sur le blocage des comptes](https://help.twitter.com/fr/using-twitter/blocking-and-unblocking-accounts) pour plus d'informations.
```
Comment (Ganondorf-1) [inline wiki/Reporting_bad_behaviour/Abuse/fr.md:58]:
```suggestion
Ensuite, bloquez ou masquez l'utilisateur en question en cliquant sur les trois points en haut à droite de son tweet (si vous utilisez le site web ou le client mobile). Consultez [le guide de Twitter sur le blocage des comptes](https://help.twitter.com/fr/using-twitter/blocking-and-unblocking-accounts) pour plus d'informations.
```

File: wiki/Reporting_bad_behaviour/Abuse/fr.md [hunk 12]
```diff
@@ -61,7 +57,7 @@ Cliquez sur les trois points en haut à droite du tweet problématique et clique
...
 
-Sachez que la personne verra que vous l'avez bloquée si elle consulte votre profil Twitter. Si vous craignez que cela n'aggrave le problème ailleurs, mettez-la plutôt en sourdine. Cela supprimera ses tweets de votre ligne de temps et vous empêchera de voir d'autres contenus de sa part, mais elle pourra toujours voir et commenter tous vos tweets.
+Sachez que la personne verra que vous l'avez bloquée si elle consulte votre profil Twitter. Si vous craignez que cela n'aggrave le problème ailleurs, mettez-la plutôt en sourdine. Cela supprimera ses tweets de votre timeline et vous empêchera de voir d'autres contenus de sa part, mais elle pourra toujours voir et commenter tous vos tweets.
```
Comment (Ganondorf-1) [inline wiki/Reporting_bad_behaviour/Abuse/fr.md:60]:
```suggestion
Sachez que la personne verra que vous l'avez bloquée si elle consulte votre profil Twitter. Si vous craignez que cela n'aggrave le problème ailleurs, masquez-la à la place. Cela supprimera ses tweets de votre timeline et vous empêchera de voir d'autres contenus de sa part, mais elle pourra toujours voir et commenter tous vos tweets.
```

File: wiki/Reporting_bad_behaviour/Handling_foul_play/fr.md [hunk 13]
```diff
@@ -1,40 +1,36 @@
...
----
-
-# Traitement d'un acte criminel
+# Traitement d'un acte malveillant
 
 ## Pourquoi suis-je restreint ?
 
 Un utilisateur peut être restreint s'il fait l'une des choses suivantes :
 
 - Un mauvais comportement continu, quel que soit le nombre d'infractions qu'ils accumulent.
```
Comment (Ganondorf-1) [inline wiki/Reporting_bad_behaviour/Handling_foul_play/fr.md:7]:
```suggestion
- Un mauvais comportement continu, quel que soit le nombre d'infractions qu'il accumule.
```

File: wiki/Reporting_bad_behaviour/Handling_foul_play/fr.md [hunk 14]
```diff
@@ -1,40 +1,36 @@
----
-no_native_review: true
----
-
-# Traitement d'un acte criminel
+# Traitement d'un acte malveillant
```
Comment (Ganondorf-1) [inline wiki/Reporting_bad_behaviour/Handling_foul_play/fr.md:1]:
```suggestion
# Traitement d'un acte de tricherie
```

File: wiki/Reporting_bad_behaviour/Handling_foul_play/fr.md [hunk 15]
```diff
@@ -1,40 +1,36 @@
...
 Un utilisateur peut être restreint s'il fait l'une des choses suivantes :
 
 - Un mauvais comportement continu, quel que soit le nombre d'infractions qu'ils accumulent.
-- Contournement d'une sanction en créant un nouveau compte.
-- Partage de compte avec d'autres utilisateurs.
+- Un contournement d'une sanction en créant un nouveau compte.
+- Un partage de compte avec d'autres utilisateurs.
 - Le multi-compte
-- Piratage ou tricherie (voir ci-dessous).
+- Du piratage ou de la tricherie (voir ci-dessous).
```
Comment (Ganondorf-1) [inline wiki/Reporting_bad_behaviour/Handling_foul_play/fr.md:11]:
```suggestion
- Du piratage ou de la triche (voir ci-dessous).
```

File: wiki/Reporting_bad_behaviour/Handling_foul_play/fr.md [hunk 16]
```diff
@@ -1,40 +1,36 @@
...
 
 ## Qu'est-ce qui est considéré comme du multi-compte ?
 
 Chaque utilisateur ne peut utiliser qu'un **un** compte. La création de nouveaux comptes est strictement interdite. La première infraction sera sanctionnée par un silence plus ou moins long en fonction du nombre de comptes créés. Les utilisateurs qui continuent à créer de nouveaux comptes malgré les avertissements précédents seront restreints.
 
 ## Qu'est-ce qui est considéré comme une tricherie ?
```
Comment (Ganondorf-1) [inline wiki/Reporting_bad_behaviour/Handling_foul_play/fr.md:17]:
```suggestion
## Qu'est-ce qui est considéré comme de la triche ?
```

File: wiki/Reporting_bad_behaviour/Handling_foul_play/fr.md [hunk 17]
```diff
@@ -1,40 +1,36 @@
...
 
-Les [règles](/wiki/Rules) stipulent que toute forme de manipulation du client, comme le piratage du score, les séquences de combo complètes activées par macro, la visée automatique et tout ce qui joue le jeu pour le joueur est considéré comme de la triche.
+Les [règles](/wiki/Rules) stipulent que toute forme de manipulation du client, comme le piratage du score, les séquences de combo complètes activées par macro, la visée automatique et tout ce qui donne des avantages de ce type est considéré comme de la triche.
 
-Il s'agit d'une règle à un coup. Dès leur première infraction, les tricheurs seront soumis à des restrictions pour une durée indéterminée. L'[Équipe de support](/wiki/People/The_Team/Account_support_team) peut décider de lever la restriction si l'appel est accepté.
+Il s'agit d'une règle "à un coup". Dès leur première infraction, les tricheurs seront soumis à des restrictions pour une durée indéterminée. L'[Équipe de support](/wiki/People/The_Team/Account_support_team) peut décider de lever la restriction si l'appel est accepté.
```
Comment (Ganondorf-1) [inline wiki/Reporting_bad_behaviour/Handling_foul_play/fr.md:21]:
```suggestion
Il s'agit d'une règle "à un coup". Dès leur première infraction, les tricheurs seront restreints pour une durée indéterminée. L'[Équipe de support aux comptes](/wiki/People/The_Team/Account_support_team) peut décider de lever la restriction si l'appel est accepté.
```

File: wiki/Reporting_bad_behaviour/Handling_foul_play/fr.md [hunk 18]
```diff
@@ -1,40 +1,36 @@
...
 
 ## J'ai été restreint pour avoir triché, mais je n'ai pas triché !
 
 osu! utilise des mesures automatisées combinées à une confirmation manuelle. Si vous ne savez pas pourquoi vous êtes restreint, contactez [accounts@ppy.sh](mailto:accounts@ppy.sh).
 
-## Prouvez que je/ils ont triché ! Dis-nous comment vous le savez !
+## Prouvez que j'ai/qu'ils ont triché(s) ! Dites-nous comment vous le savez !
 
-L'équipe de support ne fournira aucune "preuve" concernant les restrictions, mais toutes sont placées avec une suspicion bien fondée. Il est extrêmement improbable que les utilisateurs normaux soient visés par de telles actions.
+l'Équipe du support ne fournira aucune "preuve" concernant les restrictions, mais toutes sont placées avec une suspicion bien fondée. Il est extrêmement improbable que les utilisateurs normaux soient visés par de telles actions.
```
Comment (Ganondorf-1) [inline wiki/Reporting_bad_behaviour/Handling_foul_play/fr.md:29]:
```suggestion
L'équipe de support aux comptes ne fournira aucune "preuve" concernant les restrictions, mais toutes sont placées avec une suspicion bien fondée. Il est extrêmement improbable que les utilisateurs normaux soient visés par de telles actions.
```

File: wiki/Reporting_bad_behaviour/Handling_foul_play/fr.md [hunk 19]
```diff
@@ -1,40 +1,36 @@
...
 
 ## Je vais faire une vidéo pour prouver mon innocence !
 
-Désolé, mais si vous avez été restreint, l'équipe de support est déjà sûre que vous trichez. Une vidéo ne sera d'aucune utilité, car même celle-ci peut être falsifiée.
+Désolé, mais si vous avez été restreint, l'Équipe du support est déjà sûre que vous trichez. Une vidéo ne sera d'aucune utilité, car même celle-ci peut être falsifiée.
```
Comment (Ganondorf-1) [inline wiki/Reporting_bad_behaviour/Handling_foul_play/fr.md:33]:
```suggestion
Désolé, mais si vous avez été restreint, l'équipe du support est déjà sûre que vous trichez. Une vidéo ne sera d'aucune utilité, car même celle-ci peut être falsifiée.
```

File: wiki/Reporting_bad_behaviour/Handling_foul_play/fr.md [hunk 20]
```diff
@@ -43,7 +39,7 @@ Vous devez signaler l'utilisateur et ne le mentionner nulle part. Il y a trois f
 
 ### Profil de l'utilisateur
 
-**Il s'agit de la méthode préférée pour signaler les suspicions de fraude, car elle fournit le plus d'informations aux membres de l'équipe de support.**
+**Il s'agit de la méthode préférée pour signaler les suspicions de triche, car elle fournit le plus d'informations aux membres de l'Équipe du support.**
 
 Sur le profil de l'utilisateur, un bouton `Signaler` apparaît lorsque vous cliquez sur les trois points à côté du bouton `Message`. Inscrivez toutes les informations qui peuvent prouver que l'utilisateur se comporte mal.
```
Comment (Ganondorf-1) [inline wiki/Reporting_bad_behaviour/Handling_foul_play/fr.md:44]:
```suggestion
Sur le profil de l'utilisateur, un bouton `Signaler` apparaît lorsque vous cliquez sur les trois points à côté du bouton `envoyer un message`. Inscrivez toutes les informations qui peuvent prouver que l'utilisateur se comporte mal.
```

File: wiki/Reporting_bad_behaviour/Handling_foul_play/fr.md [hunk 21]
```diff
@@ -76,7 +72,7 @@ Les périphériques tels que les tablettes et les écrans tactiles sont accepté
 
 Attendez trois mois après la dernière restriction émise avant de faire appel. Adressez vos courriels à [accounts@ppy.sh](mailto:accounts@ppy.sh) et utilisez un objet du type `Restriction Appeal: <username>`
 
-L'équipe de support attend des utilisateurs qu'ils fournissent un historique complet de leurs actions et qu'ils donnent les raisons pour lesquelles ils ne récidiveront pas à l'avenir.
+l'Équipe du support attend des utilisateurs qu'ils fournissent un historique complet de leurs actions et qu'ils donnent les raisons pour lesquelles ils ne récidiveront pas à l'avenir.
```
Comment (Ganondorf-1) [inline wiki/Reporting_bad_behaviour/Handling_foul_play/fr.md:75]:
```suggestion
L'équipe du support attend des utilisateurs qu'ils fournissent un historique complet de leurs actions et qu'ils donnent les raisons pour lesquelles ils ne récidiveront pas à l'avenir.
```

File: wiki/Reporting_bad_behaviour/fr.md [hunk 22]
```diff
@@ -1,12 +1,6 @@
----
-outdated: true
-outdated_since: 2bad0ad3bfcdc272e5832ecf6d29bd61fcc6c23a
-no_native_review: true
----
-
 # Signaler un mauvais comportement
 
-osu! a [une équipe de modérateurs](/wiki/People/The_Team/Global_Moderation_Team) qui s'efforce de garder le jeu et son écosystème propres et sûrs. Si vous constatez un comportement contraire aux règles ou un contenu inapproprié, signalez-le en utilisant l'une des méthodes décrites ci-dessous. Vous pouvez également contacter directement l'un des modérateurs en ligne si votre report n'a pas été traité correctement dans un délai raisonnable, mais uniquement en dernier recours.
+osu! a [une équipe de modérateurs](/wiki/People/The_Team/Global_Moderation_Team) qui s'efforcent de garder le jeu et son écosystème propres et sûrs. Si vous constatez un comportement contraire aux règles ou un contenu inapproprié, signalez-le en utilisant l'une des méthodes décrites ci-dessous. Vous pouvez également contacter directement l'un des modérateurs en ligne si votre report n'a pas été traité correctement dans un délai raisonnable, mais uniquement en dernier recours.
```
Comment (Ganondorf-1) [inline wiki/Reporting_bad_behaviour/fr.md:3]:
```suggestion
osu! a [une équipe de modérateurs](/wiki/People/The_Team/Global_Moderation_Team) qui s'efforcent de garder le jeu et son écosystème propres et sûrs. Si vous constatez un comportement contraire aux règles ou un contenu inapproprié, signalez-le en utilisant l'une des méthodes décrites ci-dessous. Vous pouvez également contacter directement l'un des modérateurs en ligne si votre signalement n'a pas été traité correctement dans un délai raisonnable, mais uniquement en dernier recours.
```

File: wiki/Reporting_bad_behaviour/fr.md [hunk 23]
```diff
@@ -1,12 +1,6 @@
...
 
 Les reports accidentels sont ignorés. Les utilisateurs qui envoient délibérément des reports non valides risquent d'être [réduits au silence](/wiki/Silence).
```
Comment (Ganondorf-1) [inline wiki/Reporting_bad_behaviour/fr.md:5]:
```suggestion
Les signalements accidentels sont ignorés. Les utilisateurs qui envoient délibérément des reports non valides risquent d'être [réduits au silence](/wiki/Silence).
```

File: wiki/Reporting_bad_behaviour/fr.md [hunk 24]
```diff
@@ -15,7 +9,7 @@ Les reports accidentels sont ignorés. Les utilisateurs qui envoient délibéré
 | Nature de l'infraction | Point de contact |
 | :-- | :-- |
 | Messages privés offensants | **Aucun** : [ignorez l'utilisateur](/wiki/Client/Interface/Chat_console#la-liste-des-commandes). Pour un effet permanent, utilisez la liste d'ignorés du jeu située dans les [options](/wiki/Client/Options). |
```
Comment (Ganondorf-1) [inline wiki/Reporting_bad_behaviour/fr.md:11]:
```suggestion
| Messages privés offensants | **Aucun** : [ignorez l'utilisateur](/wiki/Client/Interface/Chat_console#la-liste-des-commandes). Pour un effet permanent, utilisez la liste d'utilisateurs à ignorer du jeu située dans les [options](/wiki/Client/Options). |
```

**Review by Ganondorf-1 — CHANGES_REQUESTED** [review]

File: wiki/Gameplay/Grade/fr.md [hunk 25]
```diff
@@ -1,61 +1,63 @@
...
-no_native_review: true
+tags:
+  - letter grade
+  - rank
+  - silver s
+  - sh
+  - xh
+  - lettre de grade
+  - classement
+  - s argenté
```
Comment (DoryanR) [inline wiki/Gameplay/Grade/fr.md:10]:
```suggestion
  - s argenté
  - notes
```

File: wiki/Reporting_bad_behaviour/Handling_foul_play/fr.md (hunk 14 above)
Comment (DoryanR) [reply wiki/Reporting_bad_behaviour/Handling_foul_play/fr.md:1]: J'avais du mal à trouver un titre cohérent avec l'article, c'est bien trouvé !

**Review by Biokax — APPROVED** [review]

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**